### PR TITLE
Fix hamburger menu and nav bar styling

### PR DESCRIPTION
**Pull request by aliceluo2048**
**Changes**
* Aligned hamburger menu icon
* Changed size and alignment of SCL icon in compact menu to be consistent with Weebly's compact menu
* Added styling to nav bar to be consistent with the other pages

**Before**:
https://trello-attachments.s3.amazonaws.com/5e777be61d2e4a20cf1ab0ff/5f1734ded4528611dfff357f/9157a6afe096b55f8c0d6095766e9b1d/movie.mp4

**After:**
hamburgermenu

**Before**
image
**After**
image

### CODE REVIEW

File: src/components/Logos/Logos.tsx [hunk 1]
```diff
@@ -25,7 +25,9 @@ const LogoImage = styled.img`
   height: 70px;
 
   @media (max-width: 599px) {
-    width: 112px;
-    height: 50px;
+    width: 100px;
+    height: 40px;
+    margin-left: -6px;
```
Comment (justintmckibben) [inline src/components/Logos/Logos.tsx:30]: negatives margins is a bit of code smell 😬

File: src/components/Navbar/NavBar.tsx [hunk 2]
```diff
@@ -245,7 +250,11 @@ const LanguageContainer = styled.div`
   font-size: 13px;
   width: 78px;
   margin-left: 20px;
-  ${(props: CompactProps) => props.compact === 'true' && 'margin-right: 20px;'}
+  margin-top: -4px;
```
Comment (justintmckibben) [inline src/components/Navbar/NavBar.tsx:253]: merp

**Comment by justintmckibben** [comment]
Thanks for fixing this!

File: src/components/Logos/Logos.tsx (hunk 1 above)
Comment (aliceluo2048) [reply src/components/Logos/Logos.tsx]: Yeah it's not ideal and I wanted to make it as close as the weedle's menu. The changes was so the ENG | 中文 would align with the nav links on desktop.

**Pull request merged**